sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -84,8 +84,9 @@ mod checked {
         pub gas_charger: &'a mut GasCharger,
         /// Additional transfers not from the Move runtime
         additional_transfers: Vec<(/* new owner */ SuiAddress, ObjectValue)>,
+        /// TODO(vm-rewrite): see about removing this
         /// Newly published packages
-        new_packages: Vec<MovePackage>,
+        pub new_packages: Vec<MovePackage>,
```

Comment:
Please no `pub` here. Bad things can happen if this is `pub`--very delicate code